wp-content/themes/redmond/web/scss/modules/_hero-header.scss

```diff
@@ -7,6 +7,8 @@
     padding: rem-calc(170px) 0;
     color: $primary-light-color;
     margin-bottom: rem-calc(40px);
```

Comment:
remove the bottom margin. the spacing for the row below is done through padding in that row.